addons/web/static/lib/moment/locale/km.js

```diff
@@ -15,7 +15,7 @@ var km = moment.defineLocale('km', {
     monthsShort: 'មករា_កុម្ភៈ_មីនា_មេសា_ឧសភា_មិថុនា_កក្កដា_សីហា_កញ្ញា_តុលា_វិច្ឆិកា_ធ្នូ'.split('_'),
     weekdays: 'អាទិត្យ_ច័ន្ទ_អង្គារ_ពុធ_ព្រហស្បតិ៍_សុក្រ_សៅរ៍'.split('_'),
     weekdaysShort: 'អាទិត្យ_ច័ន្ទ_អង្គារ_ពុធ_ព្រហស្បតិ៍_សុក្រ_សៅរ៍'.split('_'),
-    weekdaysMin: 'អាទិត្យ_ច័ន្ទ_អង្គារ_ពុធ_ព្រហស្បតិ៍_សុក្រ_សៅរ៍'.split('_'),
+    weekdaysMin: 'អា_ច_អ_ពុ_ព្រ_សុ_ស'.split('_'),
```

Comment:
Shouldn't this be fixed in the upstream library: https://github.com/moment/moment/blob/develop/locale/km.js#L18 ?

Reply:
Yes, exactly, please report the changes upstream and, if accepted, we will update our locales

Reply:
https://github.com/moment/moment/issues/4353